pkg/daemon/ceph/client/filesystem_mirror.go

```diff
@@ -212,8 +213,16 @@ func GetFSMirrorDaemonStatus(context *clusterd.Context, clusterInfo *ClusterInfo
 	// Using Debug level since this is called in a recurrent go routine
 	logger.Debugf("retrieving filesystem mirror status for filesystem %q", fsName)
 
+	// If cephVersion >= 16.2.6, we run `ceph fs snapshot mirror daemon status`
+	// otherwise we run `ceph fs snapshot mirror daemon status <fs_name>`
+	cephVersion := clusterInfo.CephVersion.IsAtLeast(cephver.CephVersion{Major: 16, Minor: 2, Extra: 6})
+	var args []string
+	if cephVersion {
```

Comment:
I think this reads better (and closer to an english language sentence) if the check above is part of the if statement. If the line needs to be shorter, we can do the below

```go
cephVersion := clusterInfo.CephVersion
if cephVersion.IsAtLeast(X) {

}
```